src/coreclr/jit/emit.cpp

```diff
@@ -3877,7 +3888,7 @@ void emitter::emitDispGCinfo()
 //------------------------------------------------------------------------
 // emitDispJumpList: displays the current emitter jump list
 //
-void emitter::emitDispJumpList()
+void emitter::emitDispJumpList(bool showJumpRemovalCndidates)
```

Comment:
What is the need of `showJumpRemovalCndidates`?

Reply:
To be able to see jump candidates when they're relevant (calls from 'emitRemoveJumpToNextInst`) and not see them when they aren't relevant (call from `emitJumpDistBind`). Though I think I'll add that missing `a` to make correct the spelling.

Reply:
> I don't think it is worth adding a parameter for that.

I felt it made the jit dumps more readable. Something they really need. I'll remove it.